src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -96,7 +96,7 @@ internal sealed class TracerProviderSdk : TracerProvider
 
         foreach (var instrumentation in state.Instrumentation)
         {
-            if (instrumentation.Instance != null)
+            if (instrumentation.Instance is not null)
```

Comment:
The reason for this change was not immediately obvious to me. Just commenting in case this catches anyone else's eye:

Using `!=` may case an overridden operator to be called. Using `is not null` will not.